docs/changelog.rst

```diff
@@ -92,22 +92,22 @@ Features - 4.0.0a8
 - Add support for ``{:}`` substitution in ini files as placeholder for the OS path separator - by :user:`gaborbernat`. (`#2125 <https://github.com/tox-dev/tox/issues/2125>`_)
 - When cleaning directories (for tox environment, ``env_log_dir``, ``env_tmp_dir`` and packaging metadata folders) do not
   delete the directory itself and recreate, but instead just delete its content (this allows the user to cd into it and
-  still be in a valid folder after a new run) - by :user:`gaborbernat`. (`#2139 <https://github.com/tox-dev/tox/issues/2139>`_)
+  still be in a valid folder after a new run) - by :user:`gaborbernat`. (`#2130 <https://github.com/tox-dev/tox/issues/2130>`_)
```

Comment:
This looks like the wrong number. #2139 looks right for me.

Reply:
This observation here is correct.

Reply:
Seems already fixed on rewrite branch.